lib/ai/model/agent.go

```diff
@@ -28,10 +31,11 @@ import (
 )
 
 const (
-	actionFinalAnswer = "Final Answer"
-	actionException   = "_Exception"
-	maxIterations     = 15
-	maxElapsedTime    = 5 * time.Minute
+	actionFinalAnswer   = "Final Answer"
+	actionException     = "_Exception"
+	maxIterations       = 15
+	maxElapsedTime      = 5 * time.Minute
+	finalResponseHeader = "<FINAL RESPONSE>"
```

Comment:
Can you please add Go docs here? These constants are private, but I don't think they are self-explanatory, so adding comments can make the code easier to understand.

Reply:
Solid